app/src/main/res/values/strings.xml

```diff
@@ -477,4 +477,9 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="wont_save_min_length">Files with fewer than %d characters not saved automatically in order to prevent data loss.</string>
     <string name="advanced_filtering_help"><![CDATA[<small><a href="https://www.github.com/gsantner/markor/discussions/1940">Advanced filtering syntax</a></small>]]></string>
     <string name="filter">Filter</string>
+    <string name="text_case">Text Case</string>
+    <string name="toggle_case">Toggle Case</string>
```

Comment:
Thanks,

Suggestion, put a example at each like  AbC -> aBc, abc -> ABC

Reply:
Hello, in which file should I add the examples? I can add them into TextCasingUtils.java, DocumentEditAndViewFragment.java, or HighlightingEditor.java. For example I can add the following comments to TextCasingUtils.java:
```
package net.gsantner.markor.util;

public class TextCasingUtils {
    public static String toggleCase(String text) { // Toggle all text to Uppercase or Lowercase. Ex: "Case" -> "CASE" -> "case"
        if (text.equals(text.toUpperCase())) {
            return text.toLowerCase();
        } else {
            return text.toUpperCase();
        }
    }

    public static String switchCase(String text) { // Switch the text case of each character. Ex: "CaSe" -> "cAsE"
        StringBuilder result = new StringBuilder();
        for (char c : text.toCharArray()) {
            if (Character.isUpperCase(c)) {
                result.append(Character.toLowerCase(c));
            } else if (Character.isLowerCase(c)) {
                result.append(Character.toUpperCase(c));
            } else {
                result.append(c);
            }
        }
        return result.toString();
    }

    public static String capitalizeWords(String text) { // Capitalize the first letter of each word. Ex: "this is a test." -> "This Is A Test.
        StringBuilder result = new StringBuilder();
        boolean capitalizeNext = true;
        for (char c : text.toCharArray()) {
            if (Character.isWhitespace(c)) {
                capitalizeNext = true;
                result.append(c);
            } else if (capitalizeNext) {
                result.append(Character.toUpperCase(c));
                capitalizeNext = false;
            } else {
                result.append(Character.toLowerCase(c));
            }
        }
        return result.toString();
    }

    public static String capitalizeSentences(String text) { // Capitalize the first letter of each sentence. Ex: "sentence 1. Sentence 2! sEntence 3? enD." -> "Sentence 1. Sentence 2! SEntence 3? EnD."
        StringBuilder result = new StringBuilder();
        boolean capitalizeNext = true;
        for (char c : text.toCharArray()) {
            if (c == '.' || c == '!' || c == '?') {
                capitalizeNext = true;
                result.append(c);
            } else if (Character.isWhitespace(c)) {
                result.append(c);
            } else if (capitalizeNext) {
                result.append(Character.toUpperCase(c));
                capitalizeNext = false;
            } else {
                result.append(c);
            }
        }
        return result.toString();
    }
}
```

Reply:
Display text (strings.xml)

Reply:
Oh I see, thank you. I added an example to the display text of each text case now.